src/Entity/UserMention.php

```diff
@@ -5,28 +5,37 @@
 class UserMention extends AbstractEntity
 {
     /**
-     * {@inhertDoc}.
+     * @return string
```

Comment:
How come you're explicitly settings the return type here and not relying on {@inheritdoc}?